src/test.py

```diff
@@ -65,6 +66,55 @@ def work():
 
     run_in_teamcity_block("webapp_integration_tests", work)
 
+def start_orderly_web():
+    def work():
+        ow_migrate_image = get_image_name("orderlyweb-migrate", "master")
+        pull(ow_migrate_image)
+        run([
+            "docker", "run", "--rm",
+            "-v", "demo:/orderly",
+            ow_migrate_image
+        ], check=True)
+
+        ow_cli_image = get_image_name("orderly-web-user-cli", "master")
+        pull(ow_cli_image)
+
+        #user for api blackbox tests
+        run([
```

Comment:
Can we pull these repeated blocks out into their own functions, `add_user` and `grant_permissions`?